module/documents/actor/trait.mjs

```diff
@@ -232,24 +369,73 @@ export function keyLabel(trait, key) {
 
 /**
  * Create a human readable description of the provided choice.
- * @param {string} trait        Trait as defined in `CONFIG.DND5E.traits`.
- * @param {TraitChoice} choice  Data for a specific choice.
+ * @param {TraitChoice} choice             Data for a specific choice.
+ * @param {object} [options={}]
+ * @param {boolean} [options.only=false]   Is this choice on its own, or part of a larger list?
+ * @param {boolean} [options.final=false]  If this choice is part of a list of other grants or choices,
+ *                                         is it in the final position?
  * @returns {string}
  */
-export function choiceLabel(trait, choice) {
-  // Select from any trait values
-  if ( !choice.pool ) {
-    return game.i18n.format("DND5E.TraitConfigChooseAny", {
-      count: choice.count,
-      type: traitLabel(trait, choice.count).toLowerCase()
+export function choiceLabel(choice, { only=false, final=false }={}) {
+  if ( !choice.pool.size ) return "";
+
+  // Single entry in pool
+  // { count: 3, pool: ["skills:*"] } -> any three skills
+  // { count: 3, pool: ["skills:*"] } (final) -> three other skills
```

Comment:
I might need a bit of elaboration of how the `final` option is supposed to work. Is it only for the purposes of formatting the list for display?

Reply:
Yes, it is to format lists properly when there is a mixture to grants & choices like `"Athletics and three other skills"`, which reads better than `"Athletics and any three skills"`.